packages/core/src/application.ts

```diff
@@ -248,6 +260,16 @@ export class Application extends Context {
     }
   }
 
+  private async _renderOpenApiSpec(
+    request: ServerRequest,
+    response: ServerResponse,
+  ) {
+    response.statusCode = 308;
+    response.setHeader('Location', 'http://petstore.swagger.io/?url='
+    + 'http://' + request.headers.host + '/swagger.json');
```

Comment:
http://loopback.io/doc/en/contrib/style-guide.html#one-argument-per-line or format the code using Prettier.